storage/byodb/external-buckets/index.md

```diff
@@ -93,10 +96,10 @@ WHERE `source` = "mql";
 (`sales_emea`). However, there are no technical limitations; they can have the same name. 
 
 ## Sharing an External Dataset
-It is possible to share a Snowflake external bucket using the same process as [any other Storage bucket](https://help.keboola.com/catalog/#enable-sharing). Once the bucket is shared, the refresh operation is only available in the source project. Currently, it is possible to share entire buckets, not specific tables within them.
+It is possible to share a Snowflake external dataset using the same process as [any other Storage bucket](https://help.keboola.com/catalog/#enable-sharing). Once the bucket is shared, the refresh operation is only available in the source project (the project where the external dataset has been registered in). Currently, it is possible to share entire buckets, not specific tables within them.
```

Comment:
tady je priklad toho, jak pojem "external dataset" je strasne pitomej. dataset v keboole neexistuje. Ale external dataset ano, ale on je to bucket... 💥 imho bychom ale meli drzet teda pojem external dataset, kdyz uz jsme external bucket vyradili ze slovniku